uservice_ccutter/server.py

```diff
@@ -32,7 +32,7 @@ def server(run_standalone=False):
     """
     # Add "/ccutter" for mapping behind api.lsst.codes
     app = APIFlask(name="uservice-ccutter",
-                   version="0.0.3",
+                   version="0.0.4",
```

Comment:
Yeah, I don't have a good way to simultaneously keep the code, setuptools, and Dockerfile all happy.  Same with putting requirements into setup.py rather than requirements.txt.  It's a mess and I don't think there's a perfect solution.